makefile

```diff
@@ -117,8 +117,20 @@ DEBUG_SET_ENV_VARS := \
 	export COMPANIES_HOUSE_CLIENT_SECRET=debug-client-secret; \
 	export SECURE_HSTS_SECONDS=0; \
 	export PYTHONWARNINGS=all; \
-	export PYTHONDEBUG=true
-
+	export PYTHONDEBUG=true; \
+	export EXPORTING_NEW=/new; \
+	export EXPORTING_OCCASIONAL=/occasional; \
```

Comment:
All these env vars also need adding to https://github.com/uktrade/directory-ui-export-readiness/pull/51/files#diff-cd5dbf3629c558b0104ba8f0937d6816L36 with a prefix `DIRECTORY_UI_EXPORT_READINESS_` - annoying, but needed for tests

Reply:
okay. will fix